src/examples/colors/analyze_data.py

```diff
@@ -1,22 +1,27 @@
-
-
-#End to end analysis of the WCS color data. 
-#Start here to run the full analysis of the WCS color data.
+# End to end analysis of the WCS color data.
+# Start here to run the full analysis of the WCS color data.
 from pandas import DataFrame
 import os
 from generate_wcs_languages import generate_color_languages
 from complexity import generate_color_complexity, generate_ib_bound
-from graph_colors import graph_complexity, graph_expression,graph_language_color_distribution
+from graph_colors import (
+    graph_complexity,
+    graph_expression,
+    graph_language_color_distribution,
+)
 import pandas
 import util
 
-GENERATE_IB_BOUND=False #True to generate the IB bound for the specified parameters
-USE_RKK = False #Whether to use the RKK metric for complexity
+GENERATE_IB_BOUND = False  # True to generate the IB bound for the specified parameters
```

Comment:
Those values actually get set via `argparse` in `util.py`, similar to Nathaniel's implementation. I'll remove them since they no longer need to be constants.